ReleaseNotes.md

```diff
@@ -1,6 +1,61 @@
+    * After selecting a preset, automatically highlight most commonly edited parameter (fixes #1488). So, for example, after selecting a body tube preset the Length parameter will be selected by default.
+    * Restore focus to motors, recovery, stage and simulation tables after table action and others (fixes #1558).  Simply put, you shouldn't need to click in the table areas just to enable keyboard shortcuts to work.
+    * Use tab and arrow keys to traverse sim table (fixes #1552)
+  * Show markers for pods and boosters in rocket side view (fixes #1146)
+  * Apply preset after double-clicking (fixes #1539)
+  * Pre-check diameter filters in preset chooser (fixes #1480)
+  * Preset selection window opens larger, and remembers column widdths (fixes #1305).  Separate column widths are remembered for each component type.
+  * Ctrl/Cmd+A (select all) keyboard shortcut in component tree, simulation and motor configuration tables (fixes #1549)
+
+## Notable Bug Fixes
+* **Scripting now works again.  Please test!** (fixes #308, #826, #1108, and #1270)
+* Don't re-run simulations when new sim is created with same motor (fixes #1510)
+* Ensure simulations finish when running from scripts (fixes #1575)
+* Fixed Cd calculation for finsets (fixes #1440)
+* Move component config window back to same monitor as main app window first time it is opened.  No more lost config windows! (fixes #1470)
+* Rocket side view updates
+  * Don't recenter rocket when zooming (fixes #1464)
+  * Ensure full rocket is always visible and positioned correctly in rocket side view (fixes #1465)
+* Fixed root edge display in 3D view and fin templates (fixes #1227).  This matters when you are attaching fins to a nose cone or transition.
+* Better register of double-clicks in 3D view (fixes #1054)
+* Increased component analysis drag precision to three decimal digits (fixes #1476)
+* Don't include inactive stages in calculations (fixes #1477 and #1460)
+* Select recovery device/stage after config panel select (fixes #1490)
+* Fixed an exception when scaling freeform fins (fixes #1520)
+* Improved performance of freeform finset editor (fixes #1533)
+* Better automatic calculation of fin tab sizes (fixes #1600)
+
+## Other
+* Don't open motor selection dialog when duplicating motor configurations (fixes #1555)
+* Inner tubes can have custom inner/outer appearance (fixes #1509)
+* Added sliders to Photo Studio settings (fixes #1524)
+* Added and/or improved hover texts on a bunch of different buttons and controls
+* A bunch of updated component icons
+* Added icons to most buttons
+* Improvements to Undo behavior when adding new components (fixes #1513)
+* Improvements to scale dialog (fixes #411)
+* Removed "show all compatible" option in preset chooser (fixes #1405)
+* Cleaned up several of the example rockets
+* Added warning to launch preferences that changes only affect new sims (fixes #1497)
+* More Russian translation updates
+
```

Comment:
Put something about PR #1604, which is that when the auto (packed) diameter checkbox is checked, and the parent diameter changes, that the (packed) length of a mass component dynamically updates in order to preserve the (packed) volume from before the diameter change. Is useful e.g. when you have determined a parachute's volume, which should in theory remain constant even if you change the parent's diameter; you would just pack it differently.